internal/uvm/memory.go

```diff
@@ -0,0 +1,45 @@
+package uvm
+
+import (
+	"context"
+	"fmt"
+
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+)
+
+const (
+	bytesPerPage = 4096
+	bytesPerMB   = 1024 * 1024
+)
+
+// UpdateMemory makes a call to the VM's orchestrator to update the VM's size in MB
+func (uvm *UtilityVM) UpdateMemory(ctx context.Context, sizeInMB uint64) error {
```

Comment:
I think we round memory up to an even integer when creating the UVM. We should probably do the same thing here.

Reply:
Good point. Will do.

Reply:
Done.